.github/workflows/js-css-linting.yml

```diff
@@ -0,0 +1,42 @@
+name: JavaScript and CSS Linting
+
+on:
+  pull_request:
+  push:
+    branches: [trunk]
+
+jobs:
+  check:
+    name: All
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Cache node modules
+      uses: actions/cache@v2
+      env:
+        cache-name: cache-node-modules
+      with:
+        # npm cache files are stored in `~/.npm` on Linux/macOS
+        path: ~/.npm
+        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-build-${{ env.cache-name }}-
+          ${{ runner.os }}-build-
+          ${{ runner.os }}-
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+
+    - name: Npm install and build
+      run: |
+        npm install
```

Comment:
If we don't need to do `build` then we can skip the `install` step here because `bradennapier/eslint-plus-action` triggers install automatically if `node_modules` folder is not found.

Reply:
I left `npm install` in place because it [looks like the action will use node 14](https://github.com/bradennapier/eslint-plus-action/blob/972d934f4b5b41945c4f489dc49717c80eb8be4c/Dockerfile#L1) and there could be some problems with that version (we currently are pinned to node 12 - LTS).

Reply:
Nice. LGTM!

Reply:
But if we're not building, we're doing static linting, in which node version is not needed, I don't see why the version difference would play a role here.

Reply:
True. For now I'd prefer to be explicit about installing with the version used for the project. The action will not do a build if `node_modules` is found so it's not really a problem to have it this way.